.github/workflows/openvino_code.yml

```
@@ -0,0 +1,65 @@
+name: OpenVINO Code Extension Workflow
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - 'modules/openvino_code/**'
+      - '.github/workflows/openvino_code.yml'
+
+concurrency:
+  group: ${{ github.head_ref || github.ref_name }}
```

```suggestion
  group: ${{ github.head_ref || github.ref_name }}-openvino-code
```

That would future-proof this pipeline if other pipelines are implemented with `concurrency`. `github.head_ref` and `github.ref_name` are not unique for the workflow.

---

Fixed